widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetController.kt

```diff
@@ -86,17 +90,23 @@ internal class EntryWidgetController @JvmOverloads constructor(
 
         val messagingOrDefault: (default: List<EntryWidgetContract.ItemType>) -> List<EntryWidgetContract.ItemType> = { default ->
             if (isAuthenticatedUseCase() && hasOngoingSC)
-                listOf(EntryWidgetContract.ItemType.Messaging(unreadMessagesCount), EntryWidgetContract.ItemType.ProvidedBy)
+                listOf(EntryWidgetContract.ItemType.Messaging(unreadMessagesCount))
             else
                 default
         }
 
-        return when (queuesState) {
+        val items = when (queuesState) {
             QueuesState.Empty -> messagingOrDefault(emptyState)
             QueuesState.Loading -> messagingOrDefault(loadingState)
             is QueuesState.Error -> messagingOrDefault(errorState)
             is QueuesState.Queues -> mapEntryWidgetQueues(queuesState.queues, unreadMessagesCount, hasOngoingSC)
         }
+
+        if (view.whiteLabel) {
+            return items.sorted()
+        }
+
+        return items.plus(EntryWidgetContract.ItemType.PoweredBy).sorted()
```

Comment:
I expected to see vice-a-versa:
```
if (white label enabled) {
   add PoweredBy item to the bottom and return
}

otherwise - just return
```
Am I missing anything?

Reply:
![image](https://github.com/user-attachments/assets/aabb9b2d-c738-44e4-ac6e-3914b7c15326)
I just copied the behavior from the other parts of SDK.

Reply:
Hm, maybe 'whiteLabel true' means 'hide while label. And 'whiteLabel false' means 'do not hide it'? 🤔

Reply:
☝️ Thats true. This confused me at first so much that I had to make it supper visual int [this docs](https://glia.atlassian.net/wiki/spaces/ENG/pages/3390898324/White+label+configuration+in+remote+configurations)

Reply:
But from the integrator's perspective, it sounds correct. So, white labeling from their side means that the feature will be present with their own branding.
`White label products are sold by retailers with their own branding and logo but the products themselves are manufactured by a third party`

Reply:
Oh, now I understand. It's correct then. The best we can do is to add a comment to this code for our grandchildren who will also look into this code...

Reply:
Why not?
```suggestion
        if (!view.whiteLabel) {
            items.plus(EntryWidgetContract.ItemType.PoweredBy)
        }

        return items.sorted()
```

Reply:
makes sense